server/requirements.txt

```diff
@@ -2,6 +2,5 @@ gunicorn==19.7.1
 honcho==1.0.1
 newrelic>=2.66,<2.67
 
--e git+git://github.com/superdesk/superdesk-core.git@f38aaee#egg=Superdesk-Core
--e git+git://github.com/superdesk/superdesk-planning.git@master#egg=superdesk-planning
-
+-e git+git://github.com/superdesk/superdesk-core.git@v1.32.5#egg=Superdesk-Core
```

Comment:
you can use `superdesk-core>=1.32.5,<1.33` for this one. but maybe it should just use latest stable